portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/commentServ")
+public class CommentServlet extends HttpServlet{
+    @Override
+
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+     List<Comment> loadComments = new ArrayList<>();
+
+     Query query = new Query("Comment").addSort("TimeStamp", SortDirection.ASCENDING);
+     PreparedQuery results = datastore.prepare(query);
+     System.out.println("number of elements: " + results.countEntities());
```

Comment:
Did you leave the logging statement intentionally? (sometimes the answer is yes! But it's still worth highlighting in a code review in case it was accidental).

Reply:
removed